test/bulk-update/document-manager/document-manager.test.js

```diff
@@ -71,7 +71,7 @@ describe('DocumentManager', () => {
 
     it('loads a mock file');
 
-    it('loads a draft file', async () => {
+    it.skip('loads a draft file', async () => {
```

Comment:
Why the skip? If it needs more work, do we have a ticket for that?

Reply:
Occasionally this test would time out, but I updated these tests with dependency injection for fetch so no more skip needed.